app/models/profile.rb

```diff
@@ -9,6 +11,4 @@ def self.define_store_accessors!
   end
 
   define_store_accessors!
-
-  validates :data, presence: true
```

Comment:
I removed (a) there's a default of `{}` set on the DB level and (b) this default doesn't play well with the `presence?` checl that this validation performs.